explainer.md

```diff
@@ -113,6 +144,13 @@ interface PictureInPictureWindow {
   video.addEventListener('leavepictureinpicture', function() {
     // Video element left Picture-In-Picture mode.
   });
+
+  video.addEventListener('pictureinpicturecontrolsclick', event => {
+    switch (event.id) {
```

Comment:
If we add the `id` property to the event, we should probably define the interface for the event too.

Reply:
I think @mounirlamouri meant having a dedicated `PictureInPictureControlsEventHandler` which inherits from `EventHandler` and has a `id` property.

Reply:
Updated with `PictureInPictureControlsEventHandler`. I can see the dedicated `EventHandler` behaviour be determined by the internal blink implementation; is that correct, or should there be something else here? Thanks!

Reply:
Update: `s/onpictureinpicturecontrolsclick/onpictureinpicturecontrolclick/g`